javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+
+let i = 4;
+let j = 1000;
+
+const limitedInterval = setInterval(() => {
+  $countdownDisplay.innerHTML = `${--i}`;
```

Comment:
You shouldn't use `innerHTML`, it opens you up to security attacks. You should be using `textContent` instead